conformance_pack/cloudwatch.sp

```diff
@@ -24,6 +24,7 @@ control "log_group_encryption_at_rest_enabled" {
   sql         = query.log_group_encryption_at_rest_enabled.sql
 
   tags = merge(local.conformance_pack_cloudwatch_common_tags, {
+    fedramp            = "true"
```

Comment:
@khushboo9024 For these tags in the conformance pack controls, I think we should use `fedramp_low_rev_4` and `fedramp_moderate_rev_4` instead, as `fedramp` is too broad